Translation/DatabaseFreshResource.php

```diff
@@ -2,14 +2,14 @@
 
 namespace Lexik\Bundle\TranslationBundle\Translation;
 
-use Symfony\Component\Config\Resource\ResourceInterface;
+use Symfony\Component\Config\Resource\SelfCheckingResourceInterface;
```

Comment:
Or you can branch and tag a Symfony > 2.8 compatible package. But the assetic bundle approach seems to be clean.